airflow/cli/commands/pool_command.py

```diff
@@ -60,17 +61,20 @@ def pool_get(args):
 def pool_set(args):
     """Creates new pool with a given name and slots"""
     api_client = get_current_api_client()
-    pools = [api_client.create_pool(name=args.pool, slots=args.slots, description=args.description)]
-    _show_pools(pools=pools, output=args.output)
+    api_client.create_pool(name=args.pool, slots=args.slots, description=args.description)
+    print("Pool created")
```

Comment:
Maybe add `try-except` similar to what you added in `pool_delete`?

Reply:
I would like to address this in followup PR. Probably create new decorator that will handle known errors and show only message instead of lines of traceback to make it more user friendly.